autocrop/autocrop.py

```diff
@@ -207,10 +209,15 @@ def crop(self, path_or_array):
         # ====== Actual cropping ======
         image = image[pos[0] : pos[1], pos[2] : pos[3]]
 
-        # Resize
-        image = cv2.resize(
-            image, (self.width, self.height), interpolation=cv2.INTER_AREA
-        )
+        """
+        Check if the no resize flag is given or not, if given,
+        it will not resize the image just crop it
+        """
+        if self.no_resize is not True:
+            # Resize
+            image = cv2.resize(
+                image, (self.width, self.height), interpolation=cv2.INTER_AREA
+            )
```

Comment:
Just so I understand, have you implemented the feature yet @ASHWIN990 ?

Reply:
Yes, the feature has been implemented, and i have manually checked the output image resolution, and they are as it should be.

The main issue is that, i don't know how to write the test, maybe u can help me with that, teach me most probably if possible...

Reply:
Writing the test should be pretty straightforward, but I'm not sure how comfortable you might be with unit testing and pytest. If not, here are [some great resources](https://www.freecodecamp.org/news/an-introduction-to-testing-in-python/) to [get you started](https://jacobian.org/2016/nov/27/getting-started-with-pytest/). I also found [this DataCamp course](https://www.datacamp.com/courses/unit-testing-for-data-science-in-python#!) to be quite helpful. I can help you out of course, if you have specific questions.

Background: `Cropper.aspect_ratio` is calculated from the height and width variables.

Basically, you want two tests:
1. Running `Cropper.crop(width=100, height=100, resize=True)` outputs a numpy array of size `100, 100`,
2. Running `Cropper.crop(width=100, height=100, resize=False)` outputs a numpy array whose size matches some hardcoded values you'll have gotten by running it on one of the test images. It should be square (`width == height`).